client/src/Components/CarDetails/inedx.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Rating from '@mui/material/Rating';
+import Typography from '@mui/material/Typography';
+import './Style.css';
+import PropTypes from 'prop-types';
+import seat from '../../images/Vector.png';
+import door from '../../images/door.png';
+import gear from '../../images/gear.png';
+
+function CarDetails({
+  name, rate, gearbox, type, doorNumber, seats, body,
+}) {
+  return (
+    <div className="container">
+      <Typography gutterBottom variant="h3" component="div" className="car-name">
+        {name}
+      </Typography>
+      <Rating name="read-only" sx={{ fontSize: '1.3rem' }} value={rate} readOnly />
+      <div className="box1">
+        <Typography variant="h5" gutterBottom component="div" className="attribute">
+          <img src={gear} alt="Not Found" className="image" />
```

Comment:
```suggestion
          <img src={gear} alt="Gear Icon" className="image" />
```
The alt must be descriptive, and describe the image

Reply:
done